rhino-deinst

```diff
@@ -163,16 +163,17 @@ echo "Updating package cache..."
 sudo apt-get update
 
 echo "Checking for package existence..."
+
 for i in "${desktop[@]}"; do
-  if [[ $(apt-cache search --names-only "^$i\$") ]]; then
```

Comment:
`$()` is a shell substitution, not a subshell

Reply:
It's command substitution and it runs on a subshell (https://wiki.bash-hackers.org/syntax/ccmd/grouping_subshell)  (http://mywiki.wooledge.org/BashGuide/CompoundCommands)